git-find-lfs-extensions.py

```diff
@@ -67,34 +48,34 @@ def add_file(ext, size_mb):
         result[ext]['min'] = size_mb
 
 def print_line(ext, share_large, count_large, count_all, size_all, min, max):
-    print '{}{}{}{}{}{}{}'.format(
+    print ('{}{}{}{}{}{}{}'.format(
         ext.ljust(COLUMN_WIDHT),
         str(share_large).rjust(10),
         str(count_large).rjust(10),
         str(count_all).rjust(10),
         str(size_all).rjust(10),
         str(min).rjust(10),
         str(max).rjust(10)
-    )
+    ))
 
-for root, dirs, files in os.walk(cwd):
+for root, dirs, files in os.walk(os.getcwd()):
     for basename in files:
         filename = os.path.join(root, basename)
         try:
             size_mb = float(os.path.getsize(filename)) / 1024 / 1024
             if not filename.startswith(os.path.join(cwd, '.git')) and size_mb > 0:
-                if is_binary(filename):
-                    file_type = "bin"
-                else:
-                    file_type = "txt"
                 ext = filename
                 add_file('**  all  **', size_mb)
                 while ext.find('.') >= 0:
                     ext = ext[ext.find('.')+1:]
                     if ext.find('.') <= 0:
-                        add_file(file_type + "  -  " + ext, size_mb)
-        except Exception, e:
-            print e
+                        add_file(ext, size_mb)
+                noExtFile = filename
+                if noExtFile.find('.') == -1:
```

Comment:
True, we can go with ```filename```. My purpose was to have a similar appraoch as in https://github.com/larsxschneider/git-repo-analysis/blob/master/git-find-lfs-extensions.py#L90.